packages/cicero-cli/index.js

```diff
@@ -254,4 +258,7 @@ require('yargs')
         alias: 'v',
         default: false
     })
+    .demandCommand()
```

Comment:
This code is duplicated. Please remove it.

Reply:
oops!